chart/README.md

```diff
@@ -275,6 +275,7 @@ The following tables lists the configurable parameters of the Airflow chart and
 | `webserver.nodeSelector`                              | Node labels for pod assignment                                                                               | `{}`                                              |
 | `webserver.affinity`                                  | Affinity labels for pod assignment                                                                           | `{}`                                              |
 | `webserver.tolerations`                               | Toleration labels for pod assignment                                                                         | `[]`                                              |
+| `flower.enabled`                                      | Enable flower                                                                                                | `false`                                           |
```

Comment:
This should be `True` by default. Most of the users would like to have flower when using Celery or CeleryKubernetes Executor

Reply:
> Most of the users would like to have flower when using Celery or CeleryKubernetes Executor.

You are right, but executor default is `KubernetesExecutor` in values